admin/eliotlogging.py

```diff
@@ -0,0 +1,51 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Filter Eliot logs out of Twisted test.log.
+"""
+
+import sys
+
+from flocker.testtools._base import _get_eliot_data
```

Comment:
Also looking at its implementation, and looking at a `test.log`... what would happen to a line with:

```
... [HTTP11ClientProtocol (TLSMemoryBIOProtocol),client] ELIOT: {...
```

I think it would just be dropped since it didn't have `[-]`? Seems like that method could be improved. Maybe we shouldn't use it? Another approach would be to strip off everything before the first '{' and everything after the final '}', then print the line so long as it can be parsed as JSON.